pkg/api/v1/types.go

```diff
@@ -118,12 +118,13 @@ type DatacenterMeta struct {
 // AWSSize represents a object of AWS size.
 // swagger:model AWSSize
 type AWSSize struct {
-	Name       string  `json:"name"`
-	PrettyName string  `json:"pretty_name"`
-	Memory     float32 `json:"memory"`
-	VCPUs      int     `json:"vcpus"`
-	GPUs       int     `json:"gpus"`
-	Price      float64 `json:"price"`
+	Name         string  `json:"name"`
+	PrettyName   string  `json:"pretty_name"`
```

Comment:
Can you change to camelCase? I know it's mixed but it would be good if we would be targeting only one format.

Reply:
it's already in use in the UI. This change will break displaying the name